### ISSUE TITLE
[ed fusio parser] feed_infos.txt should be mandatory in ntfs v0.7

### ISSUE THREAD

**Issue by datanel** [hove-io/navitia#2276]
related to https://github.com/CanalTP/navitia/pull/2275

`feed_infos.txt` should be mandatory in https://github.com/CanalTP/navitia/blob/dev/source/ed/connectors/fusio_parser.cpp#L1745

**Comment by eturck** [comment]
Sorry to interfere. We currently have a few ntfs export and we do not use feed_infos.txt (yet) since the documentation was stating that it's optional.
Why will it become mandatory? In the doc I only see 'ntfs_version' as a mandatory feed_info_param but it doesn't seem to be read in fusio_parser (maybe I missed it). All other feed_info_param seems optional. Will an empty feed_infos.txt would be valid then?

**Comment by pbougue** [comment]
@eturck you've been right to interfere :)
So the plan is to make it mandatory in the next NTFS version to allow people to create (or modify) the parser depending on the NTFS version.
We will probably only support the reading of the last version, but if someone wanted to add support of previous ones, it would be possible. And if someone wants to support previous NTFS version, he would have to guess the version (0.6?) if feed_info is missing.

So if you're OK : NTFS 0.6 has optional feed_info (and a warning that it's gonna be mandatory)
NTFS 0.7 will have a mandatory feed info (to be merged whenever you're ready).

**Comment by eturck** [comment]
Yes it's ok for me. It's pretty easy to modify for us anyway I was mainly just curious :).
NTFS v0.7 is coming soon? Any exciting changes or just feed_info :smiley:? Should we bother @prhod for months like we did 2 years ago to have the documentation on github :grin:?

**Comment by pbougue** [comment]
You can always bother @prhod !
But the only thing that's coming and a breaking change is that feed_info thing 😃 
It's gonna be clearer and that problem is gonna be behind, that's just the point of it.
We may get an English doc at some point,but I don't want to over-promise 😉
